src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -99,10 +112,15 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
                                                         behandlingPåVentDto.venteårsak,
                                                         behandlingPåVentDto.tidsfrist)
         oppdaterAnsvarligSaksbehandler(behandlingId)
+
+        // historikkinnslag
```

Comment:
Alle disse kommentarene om historikkinnslag ser ut som huskelapper for deg selv i programmeringsfasen. De burde fjernes, siden de ikke gir noe merverdi.